docs/topic-guides/workflows/workflow-file.rst

```diff
@@ -0,0 +1,364 @@
+.. _workflow-definition-file-topic-guide:
+
+The Renku Workflow File
+=======================
+
+If your data processing workflow has many steps, you might find it convenient to
+create a workflow definition file. A workflow file help organize your data
+processing flow, making it easier to execute portions of the pipeline at a time.
+A workflow file also makes your code easier for your collaborators to
+understand!
+
+
+Adding a Workflow File to your Project
+--------------------------------------
+
+To create a workflow definition file in your project, create a file called
+``workflow.yml``. 
+
+You may have more than one workflow file in your project! Workflow files may be
+named however you like, as long as they end with the `.yml` or `yaml` file
+extension.
+
+.. note:: **Do you already have a workflow you created on the Renku CLI that you'd like to convert to a Workflow File?**
+    You can export it! ``renku workflow export <workflow-name> --format renku --output workflow.yml``
+    (Tip: Find your workflow's name using ``renku workflow ls``)
+
+
+Defining a Basic Workflow File
+------------------------------
+
+There are a few options for how you may define your workflow.
+
+In the simplest version of the Renku workflow file, the command, inputs and
+outputs are simply listed, as in the example below:
+
+.. code-block:: yaml
+
+    name: flights-processing-pipeline
+    steps:
+      filter:
+        command: python src/filter_flights.py data/flight-data/2019-01-flights.csv.zip data/output/flights-filtered.csv
```

Comment:
I believe this is too verbose of an example for the topic guide. In the next section we also include a parameter that does not actually make sense in this context (`-n 10`) so it's confusing if the user has already done the tutorial. Can we simplify?